src/components/EmailTemplates/EmailSending.jsx

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line no-unused-vars
+import React, { useState } from 'react';
+import axios from 'axios';
+import { renderEmail } from 'react-html-email';
+//import { sendEmail } from '../../utils/utils';
+
+const AISSBackend = axios.create({
```

Comment:
⚠️ **[eslint]** <react-refresh/only-export-components> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Fast refresh only works when a file only exports components. Move your component(s) to a separate file.